src/index.ts

```diff
@@ -26,6 +27,7 @@ export class Resolver {
   private _isCaughtUpWithTopic = new Map<string, boolean>();
   private _subscriptions: (() => void)[] = [];
   private cache: ICache;
+  private _udResolver: any;
```

Comment:
we can just call this the whole thing, we dont need to abbreviate it

Reply:
I don't understand what your asking here.

Reply:
rename this to `_unstoppableDomainsResolver` we dont need to abbreviate to UD resolver in this context

Reply:
done